apps/barista-design-system/src/environments/environment.internal.ts

```diff
@@ -0,0 +1,22 @@
+/**
+ * @license
+ * Copyright 2020 Dynatrace LLC
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+export const environment = {
+  deployUrl: '/',
+  internal: true,
+  searchEndpoint: '{{BARISTA_INTERNAL_SEARCH_ENDPOINT}}',
```

Comment:
@lukasholzer When building internal version, please replace this with the correct url.